__tests__/project.test.js

```diff
@@ -136,16 +136,18 @@ describe("/api/articles/:article_id/comments", () => {
       .get("/api/articles/1/comments")
       .expect(200)
       .then(({ body }) => {
-        console.log(body, "<-- body");
-        body.forEach((element) => {
-          expect(element).toHaveProperty("comment_id");
-          expect(element).toHaveProperty("votes");
-          expect(element).toHaveProperty("created_at");
-          expect(element).toHaveProperty("author");
-          expect(element).toHaveProperty("created_at");
-          expect(element).toHaveProperty("votes");
-          expect(element).toHaveProperty("body");
-          expect(element).toHaveProperty("article_id");
+        expect(body.msg.length > 0).toBe(true);
+
+        body.msg.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
+          expect(comment.article_id).toBe(1);
```

Comment:
I know this is for a previous task, but msg is not an appropriate name for the key holding the array of comments. This should simply be comments